csv_generator/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM python:3.6-stretch
+
+# pre-requisites
+RUN apt-get update && \
+    pip install -U pipenv
+
+ARG PROJECT_HOME=/usr/local/csv-generator
+ARG PROJECT_USER=csv-generator
+WORKDIR ${PROJECT_HOME}
+
+RUN useradd -s /bin/bash -d ${PROJECT_HOME} ${PROJECT_USER}
+
+COPY Pipfile .
+COPY Pipfile.lock .
+RUN echo ""
+RUN pipenv install --system --deploy
+
+COPY . ./
+
+RUN chown -R ${PROJECT_USER}: .
+RUN mkdir -p /xml-data && chown -R ${PROJECT_USER}: /xml-data
```

Comment:
good defaults for volumes, can two `VOLUME` statements be added to declare they are such?